go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/notaryproject/notation-core-go
 
-go 1.22
+go 1.23.0
```

Comment:
We need `go 1.23.1` for [CVE-2024-34156](https://nvd.nist.gov/vuln/detail/CVE-2024-34156) ([GO-2024-3106](https://pkg.go.dev/vuln/GO-2024-3106)).

Reply:
Never mind as this is the library code.